### DISCUSSION TITLE
Resource Management for Self Hosted Runners

### DISCUSSION THREAD

**Discussion by rabbiveesh** [community/community#40557] [Actions] [answered]
### Select Topic Area

Question

### Body

Is there a way to control resource usage for self hosted runners?

I'm currently using a 24 core self hosted box for our CI, but it seems to me like only 4 or so of those cores are used by the runner. When it runs our test suite (which is supposed to use 18 cores), it takes considerably longer than when I run it directly.

Is there some option somewher to control this, or is there something else going on making it slower?

**Comment by ossanna16** [comment]
Hi there @rabbiveesh and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by rabbiveesh** [comment, marked as answer]
So, after I looked into it, the issue was with my test suite, not with the runner. It seems like there are no limits imposed by the runner software.

Thanks github!
